pkg/controllers/scheduling/schedule.go

```diff
@@ -12,12 +12,23 @@ import (
 	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
 	clusterapiv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
 	"open-cluster-management.io/placement/pkg/plugins"
+	"open-cluster-management.io/placement/pkg/plugins/addon"
 	"open-cluster-management.io/placement/pkg/plugins/balance"
 	"open-cluster-management.io/placement/pkg/plugins/predicate"
 	"open-cluster-management.io/placement/pkg/plugins/resource"
 	"open-cluster-management.io/placement/pkg/plugins/steady"
 )
 
+const (
+	PrioritizerBalance        string = "Balance"
+	PrioritizerSteady         string = "Steady"
+	PrioritizerResourcePrefix string = "Resource"
+	PrioritizerAddOnPrefix    string = "AddOn"
+
+	ScoreCoordinateTypeBuiltIn string = "BuiltIn"
```

Comment:
I think we already define this in API repo

Reply:
Will fix this in https://github.com/open-cluster-management-io/api/pull/126